File: browser/src/main/java/org/sqlunet/browser/xn/HomeFragment.java
/*
 * Copyright (c) 2023. Bernard Bou
 */

package org.sqlunet.browser.xn;

import android.os.Bundle;
import android.view.View;

import org.sqlunet.browser.R;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

/**
 * Home fragment
 *
 * @author <a href="mailto:1313ou@gmail.com">Bernard Bou</a>
 */
public class HomeFragment extends org.sqlunet.browser.HomeFragment
{
	/**
	 * Mandatory empty constructor for the fragment manager to instantiate the fragment (e.g. upon screen orientation changes).
	 */
	public HomeFragment()
	{
	}

	@Override
	public void onViewCreated(@NonNull final View view, @Nullable final Bundle savedInstanceState)
	{
		super.onViewCreated(view, savedInstanceState);

		final HomeImageView homeImage = view.findViewById(R.id.splash);
		homeImage.init();
	}
}

File: browser/src/main/java/org/sqlunet/browser/xn/HomeFragment.kt
/*
 * Copyright (c) 2023. Bernard Bou
 */
package org.sqlunet.browser.xn

import android.os.Bundle
import android.view.View
import org.sqlunet.browser.HomeFragment
import org.sqlunet.browser.R

/**
 * Home fragment
 *
 * @author [Bernard Bou](mailto:1313ou@gmail.com)
 */
class HomeFragment : HomeFragment() {

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        val homeImage = view.findViewById<HomeImageView>(R.id.splash)
        homeImage.init()
    }
}
